docs/content/configuration/creating_pvc.md

```diff
@@ -59,6 +62,25 @@ Create a PVC YAML file, similar to the following `demo-pvc-raw-block.yaml` file,
           storage: 1Gi
       storageClassName: demo-storageclass
 
+## Creating a PVC with `volume_group` defined in StorageClass
```

Comment:
this is not an example of volume_group defined in storageclass.
it is actually an example of PVC to create in volumegroup with the dynamic volume group feature

Reply:
updated